tools/building.py

```diff
@@ -269,6 +269,11 @@ def link_lld(args, target, external_symbols=None):
   if settings.STRICT:
     args.append('--fatal-warnings')
 
+  if '--strip-all' in args:
+    # Tell wasm-ld to always generate a target_features section even if --strip-all
+    # is passed.
+    args.append('--keep-section=target_features')
```

Comment:
Is there a downside to always passing this in, that is, without the `if`? That would also make things work if there are other flags that cause sections to be stripped other than `--strip-all`.

Reply:
There are no other linker flags that disable the target_features section.

The downside of always passing it is simply extra noise in the link command which I think can make debugging harder sometimes.